widgets/nuxeo-selectivity.html

```diff
@@ -513,19 +519,21 @@
       },
 
       _wrap: function(value) {
-        var fn = function(item) {
+        var fn = function(item, level) {
+          var depth = level || 0;
           var id = this.idFunction(item);
           var entry = {
             id: id,
             text: item.displayLabel || item.text || id || '',
-            item: item
+            item: item,
+            depth: depth
           };
           if (item.children) {
-            entry.children = item.children.map(fn.bind(this));
+            entry.children = item.children.map(function(child) { return fn(child, entry.depth + 1) });
           }
           return entry;
-        };
-        return Array.isArray(value) ? value.map(fn.bind(this)) : fn.call(this, value);
+        }.bind(this);
+        return Array.isArray(value) ? value.map(function(item) { return fn(item) }) : fn(value);
```

Comment:
Passing `fn` to the map should do the same:
```suggestion
        return Array.isArray(value) ? value.map(fn) : fn(value);
```

Reply:
Added a comment explaining why it does not the same 👍